app/views/compute_resources_vms/form/vmware/_base.html.erb

```diff
@@ -3,6 +3,12 @@
 <%= selectable_f f, :cpus, 1..compute_resource.max_cpu_count, { }, :class => "col-md-2", :disabled => !new_host, :label => _('CPUs') %>
 <%= selectable_f f, :corespersocket, 1..compute_resource.max_cpu_count, { }, :class => "col-md-2", :disabled => !new_host, :label => _('Cores per socket') %>
 <%= text_f f, :memory_mb, :class => "col-md-2", :disabled => !new_host, :label => _("Memory (MB)") %>
+<%= field(f, :firmware, :label => _('Firmware'), :label_size => "col-md-2") do
+  compute_resource.firmware_types.collect do |type, name|
+    radio_button_f f, :firmware, {:disabled => !new_host, :value => type,
+        :text => _(name)}
```

Comment:
> What do you mean by not wrapping these? Hardcode the options and not loop over them?

Oh no, we tend to wrap lines at 100+ column and not at 80th... no
biggie.

-- 
Later,
 Lukas #lzap Zapletal